src/CommunityToolkit.Maui.Markup.UnitTests/BindableLayoutExtensionsTests.cs

```diff
@@ -1,82 +1,83 @@
 ﻿using System;
+using CommunityToolkit.Maui.Markup.UnitTests.Base;
 using Microsoft.Maui.Controls;
 using NUnit.Framework;
 
 namespace CommunityToolkit.Maui.Markup.UnitTests;
 
 [TestFixture]
-public class BindableLayoutExtensionsTests : MarkupBaseTestFixture<StackLayout>
+class BindableLayoutExtensionsTests : BaseMarkupTestFixture<StackLayout>
```

Comment:
Test class should be public

Reply:
In NUnit, test classes don't need to be public.

I Xunit, test classes **do** need to be public.